packages/dds/sequence/src/test/fuzz/fuzzUtils.ts

```diff
@@ -529,6 +529,7 @@ export const defaultFuzzOptions: Partial<DDSFuzzSuiteOptions> = {
 	},
 	defaultTestCount: 100,
 	saveFailures: { directory: path.join(_dirname, "../../src/test/fuzz/results") },
+	testSquashResubmit: true,
```

Comment:
this is the reason for small amount of seed shuffling. I did reenable all seeds and skip only necessary ones.